apps/browser/qml/pages/components/BookmarkEditDialog.qml

```diff
@@ -97,6 +96,10 @@ UserPromptDialog {
                 EnterKey.enabled: canAccept
                 EnterKey.iconSource: "image://theme/icon-m-enter-accept"
                 EnterKey.onClicked: accept()
+
```

Comment:
Let's have a custom error for the "acceptableInput" false. I.e. 

```
                description: {
                    if (errorHighlight) {
                        if (!acceptableInput) {
                            //% "URL scheme (%1) missing"
                            return qsTrId("sailfish_browser-la-missing_url_scheme").arg(scheme)
                        }
                        //% "URL is required"
                        return qsTrId("sailfish_browser-la-url_editor_error")
                    }
                    
                    return ""
                }

```

Reply:
Done.